content/tutorials/external/GRASS_for_EO_in_Jupyter.qmd

```diff
@@ -0,0 +1,31 @@
+---
+title: "GRASS for Earth Observation data processing with Jupyter notebooks"
+date: 2024-12-02
+author: Veronica Andreo
+format: 
+  html:
+    toc: true
+    code-tools: true
+    code-copy: true
+    code-fold: false
+categories: [advanced, Python, Google Colab, Earth Observation, external]
```

Comment:
I would use "remote sensing" instead of earth observation, but I wonder if that's a US vs European/South American terminology

Reply:
Here a comparison of the terms:

https://trends.google.com/trends/explore?date=today%205-y&q=earth%20observation,remote%20sensing&hl=en